.circleci/config.yml

```diff
@@ -427,8 +431,17 @@ jobs:
           command: node build-system/pr-check/experiment-e2e-tests.js --experiment=experiment<< parameters.exp >>
       - store_test_output
       - teardown_vm
+  cut_nightly:
+    executor:
+      name: node-docker-small
+    steps:
+      - run:
+          name: '⭐ Cut Nightly Branch ⭐'
+          command: node --unhandled-rejections=strict build-system/scheduled-jobs/cut-nightly.js
```

Comment:
Curious: what other scripts do you plan on adding to this directory? Instead of `scheduled-jobs`, would it make sense to call the directory `release-workflows` or `release-scripts` or something similar?

Reply:
The plan is to add any release-related workflow here. I like `release-workflows` better than `-scripts` (of course it's a script, that's just superfluous data) - renamed